kpatch-build/create-diff-object.c

```diff
@@ -2381,8 +2445,6 @@ int main(int argc, char *argv[])
 	kpatch_mark_ignored_sections(kelf_patched);
 	kpatch_compare_correlated_elements(kelf_patched);
 	kpatch_check_fentry_calls(kelf_patched);
-	kpatch_elf_teardown(kelf_base);
-	kpatch_elf_free(kelf_base);
```

Comment:
You are right, we shouldn't change the placement of teardown of the base elf. This approch is bad.
So I will modify the code as you suggested.
Thanks.